src/gwt/src/org/rstudio/studio/client/workbench/views/terminal/TerminalSession.java

```diff
@@ -740,6 +755,11 @@ public void onError(ServerError error)
          }
       });
    }
+   
+   public void showZombieMessage()
+   {
+      writeln("[Process completed]");
```

Comment:
Good idea. I can also capture exit code and make it available via rstudioapi. I'll do this via a separate PR.